tests/manage/z_cluster/upgrade/test_check_osd_mon_post_ocs_upgrade.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+import pytest
+
+from ocs_ci.framework.testlib import (
+    post_ocs_upgrade,
+    post_ocp_upgrade,
+)
+from ocs_ci.ocs.cluster import validate_osd_pdb, CephCluster
+from ocs_ci.ocs.ocp import get_ocs_version
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.helpers import get_mon_pdb
+
+log = logging.getLogger(__name__)
+
+
+@post_ocp_upgrade
+@post_ocs_upgrade
+@pytest.mark.polarion_id("OCS-2524")
+def test_check_osd_pdb_post_upgrade():
+    """
+    Test to verify OSD PDBs
+    1. Post OCP and OCS successful upgrades check for OSD PDBs
+    2. Rook-ceph-drain-canary pods disappeared after upgrade of OCS version to ocs-operator.v4.6.2-233.ci
+
+    """
+    assert validate_osd_pdb(), "No OSD PDBs present in the cluster"
+
+    # Check for rook-ceph-canary pods
+    ocs_version = get_ocs_version()
+    assert get_all_pods(
+        selector=constants.ROOK_CEPH_DRAIN_CANARY
+    ), f"Canary pods not found on {ocs_version}"
```

Comment:
Since you're not using the ocs_version but only for the assert message, you can use get_ocs_version() in the message.
That way you don't call that function with no need.